datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -69,6 +69,7 @@ use parquet::arrow::arrow_writer::{
 use parquet::arrow::{
     arrow_to_parquet_schema, parquet_to_arrow_schema, AsyncArrowWriter,
 };
+#[allow(deprecated)]
```

Comment:
main has been updated to use the new APIs, but for the maintenance release let's just use the existing code